src/main/webapp/main/LandingPage.js

```diff
@@ -29,19 +40,31 @@ const columnsCopy = [{
 }, {
   dataField: 'status',
   text: 'Status',
-  sort: true
+  sort: true,
+  headerStyle: (column, colIndex) => {
```

Comment:
Do you need `(column, colIndex)` ? They're not used. Probably could be as simple as
```
headerStyle: { width: styles.statusWidth }
```